uiowa/uiowa
# Update admissions counselor territory options

Resolves final checkboxes of #2649.



# How to test

- `blt ds --site=admissions.uiowa.edu`
- Go to http://admissions.local.drupal.uiowa.edu/counselors
- Check that "International Recruitment" option appears at the top (below " - Any - ")
- Check that only "International Recruitment", states, and DC are included in the list (American Samoa, Guam, Northern Mariana Islands, Puerto Rico, United States Minor Outlying Islands, and Virgin Islands, U.S. have been removed)
- Check that a counselor person type has the same above options for their territory
